subsys/net/lib/openthread/platform/uart.c

```diff
@@ -159,6 +159,39 @@ otError otPlatUartEnable(void)
 	uart_irq_callback_user_data_set(ot_uart.dev,
 					uart_callback,
 					(void *)&ot_uart);
+
+#ifdef CONFIG_OPENTHREAD_COPROCESSOR_SPINEL_ON_UART_ACM
+	{
```

Comment:
Why the braces here?

Reply:
To create functional block which includes defined variables at the beginning of it which helps treat this functionality as a whole without compile/static analyzer warnings.